cowamm/profitability/competitor_kpis/curve/curve_largest_2token_pools_4232976.sql

```diff
@@ -0,0 +1,65 @@
+-- Finds all the curve pools with 2 tokens and their TVLs
+-- Input: blockchain
+
+with 
+-- filters pools with 2 tokens
+pools as(
+select pool_address as contract_address,
+coin0 as token0,
+coin1 as token1,
+mid_fee*power(10,-10) as fee
+from curvefi_{{blockchain}}.view_pools
+where coin2 = 0x0000000000000000000000000000000000000000),
+
+-- finds all transfers in and out of the pools to rebuild the reserves
+transfers as(
```

Comment:
Can't re replace the four unions by something like https://github.com/cowprotocol/dune-queries/blob/b26d06e362b2c4a6c17740d42181d88cbd872670/cowamm/kpi/tvl_by_chain_4096107.sql#L25-L40 which I believe accomplishes the same thing in half the lines